util/argo/testdata/desired_deployment.yaml

```diff
@@ -17,7 +17,12 @@ spec:
         app: guestbook-ui
     spec:
       containers:
-        - image: 'gcr.io/heptio-images/ks-guestbook-demo:0.1'
-          name: guestbook-ui
-          ports:
-            - containerPort: 80
+      - image: 'gcr.io/heptio-images/ks-guestbook-demo:0.1'
+        name: guestbook-ui
+        env:
+        - name: DUPLICATED
```

Comment:
You should add a unit test in `/util/argo/managedfields/managed_fields_test.go` so that if these two fields are removed from the testdata files, then a test failure occurs.